### ISSUE TITLE
[Other] Publish an official image to docker hub?

### ISSUE THREAD

**Issue by dominicbarnes** [jubos/fake-s3#225]
I love this project, and I'm trying to use it locally during development. Unfortunately, there is no official docker image that is being published, and all the others are painfully out of date (usually >1 year old).

I noticed there's already a `Dockerfile` that is used here for running tests, any chance that could also be pushed to docker hub?

**Comment by jubos** [comment]
Any reason you need Docker vs. just installing with ruby gems?  Fake-S3 in general is meant to have almost 0 dependencies other than just the ruby stdlib, so it can easily run without the overhead of docker.

**Comment by dominicbarnes** [comment]
I don't use Ruby anywhere in my stack, and using docker is a nice way to abstract away all the dependencies for things when I'm just developing something locally.

Looking over that `Dockerfile` more, it's clearly just for running tests locally, not for any sort of distribution, so I'll close this issue.

**Comment by ahfeel** [comment]
I agree with @dominicbarnes, this would really really be helpful. In our development environment that is based on Docker, we're currently relying on unofficial docker images and most of them are out of date. Having an official Docker image would be awesome.

**Comment by jubos** [comment]
I will leave this open and investigate.

**Comment by tspecht** [comment]
Would also really love this! I think for everyone using docker already this makes a lot of sense since running FakeS3 in it's own container basically acts like the "real" S3 service in production and can be easily wired-up using a connection URL. Another user already published FakeS3 as a docker image on Dockerhub (although it's quite outdated by now ...) so maybe the Dockerfile he used could be seen as a starting-point to ease things up: https://hub.docker.com/r/lphoward/fake-s3/~/dockerfile/

**Comment by araker** [comment]
+1 for a official docker image. Here is another example of a docker file, it uses alpine linux instead of debian, which makes the image 27 Mb instead of 161 Mb. 

https://gist.github.com/araker/3b106ea9bd324edc9084388ec0346851

**Comment by razor-x** [comment]
I have maintain and use this one for personal and professional work. https://hub.docker.com/r/ourtownrentals/fake-s3/

**Comment by danihodovic** [comment]
@razor-x can you setup daily builds on Docker hub so that the image is kept up to date?
